example/next/ui/chain-switcher.tsx

```diff
@@ -17,6 +18,9 @@ export const ChainSwitcher: FC = () => {
     },
   });
 
+  const mainnetChainsArray = getChainDataArray(["cosmoshub", "juno", "sommelier", "stargaze", "osmosis"]);
+  const { osmosistestnet } = getChainData("osmosistestnet");
```

Comment:
Put outside component to prevent data refetching (beteween imports and component declaration)